vendor/blockchain_contracts/src/rfc003/mod.rs

```diff
@@ -0,0 +1,4 @@
+pub mod seconds;
+pub mod secret;
+pub mod secret_hash;
+pub mod timestamp;
```

Comment:
❓ shouldn't this stuff be private? Why is it `pub`.

Reply:
can't help it